src/Files.App/Views/Pages/Properties.xaml.cs

```diff
@@ -285,5 +363,22 @@ public class PropertyNavParam
 			public object navParameter;
 			public IShellPage AppInstanceArgument { get; set; }
 		}
+		#endregion
+	}
+
+	public class SquareNavViewItem : ObservableObject
+	{
+		public string Name;
+
+		public string Tag;
+
+		public string GlyphPrimary;
+
+		public string GlyphSecondary;
+
+		public bool UseCustomGlyph;
+
+		private bool _isSelected;
+		public bool IsSelected { get => _isSelected; set => SetProperty(ref _isSelected, value); }
 	}
-}
```

Comment:
I think you should split that in multiple lines

Reply:
Should not use too much lines. When those code will have been moved to ViewModel, I will.